### PR TITLE
feat: add resource requests/limits to postgres-package

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Comment by barnabasbusa** [comment]
Related issue: https://github.com/kurtosis-tech/kurtosis/issues/1700

Would like to be able to test out this package without needing to modify kurtosis.yaml

File: kurtosis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: "github.com/kurtosis-tech/postgres-package"
+name: "github.com/barnabasbusa/postgres-package"
```
[the remark this answers is not in the document]
Comment (barnabasbusa) [reply kurtosis.yml:1]: https://github.com/kurtosis-tech/kurtosis/issues/1700 well tell that to kurtosis devs lmao

File: kurtosis.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 name: "github.com/kurtosis-tech/postgres-package"
 description: |
   # Postgres
-  This is the Kurtosis package for launching a Postgres instance. It also contains configuration parameters to help get going with the instance; see the `main.star#run` function for details.
+  This is the Kurtosis package for launching a Postgres instance. It also contains configuration parameters to help get going with the instance; see the `main.star#run` function for details.
```
Comment (h4ck3rk3y) [inline kurtosis.yml:4]: can we add it back?

**Review by h4ck3rk3y — APPROVED** [review]
lets go

**Pull request merged**